src/tcgwars/logic/impl/gen8/DarknessAblaze.groovy

```diff
@@ -2854,8 +2869,10 @@ public enum DarknessAblaze implements LogicCardInfo {
             assert my.deck : "Your deck is empty"
           }
           onAttack {
-            my.deck.search(max:2,"Select up to 2 cards",{true}).moveTo(hidden:true,my.hand)
-            shuffleDeck()
+            if (my.deck) {
```

Comment:
Won't this also always be true because of the assert in `attackRequirement`?

Reply:
Also not if the attack is copied by another attack/ability.